packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  ChainId,
+  gweiDecToWEIBN,
+  hexToBN,
+  query,
+  toHex,
+} from '@metamask/controller-utils';
+import type EthQuery from '@metamask/eth-query';
+import type { GasFeeEstimates } from '@metamask/gas-fee-controller';
+import { GAS_ESTIMATE_TYPES } from '@metamask/gas-fee-controller';
+import { createModuleLogger, type Hex } from '@metamask/utils';
+import type { BN } from 'ethereumjs-util';
+
+import { projectLogger } from '../logger';
+import type {
+  GasFeeFlow,
+  GasFeeFlowRequest,
+  GasFeeFlowResponse,
+  TransactionMeta,
+} from '../types';
+import { DefaultGasFeeFlow } from './DefaultGasFeeFlow';
+
+type LineaEstimateGasResponse = {
+  baseFeePerGas: Hex;
+  priorityFeePerGas: Hex;
+};
+
+type FeesByLevel = {
+  low: BN;
+  medium: BN;
+  high: BN;
+};
```

Comment:
Throughout this file we could replace low, medium and high with an enum:

```typescript
FeeByPriority {
    [key: Priority]: BN,
}
```

Reply:
Done.